ui-tests/src/test/java/org/openmrs/reference/page/EditPatientRelationshipPage.java

```diff
@@ -0,0 +1,34 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.Select;
+
+public class EditPatientRelationshipPage  extends Page {
+
+    private  HomePage homePage;
+    private static String name = "john";
+    private  static  final By SELECT_RELATIONSHIP_TYPE = By.id("relationship_type");
+    private static final By NAME = By.cssSelector("button, input");
```

Comment:
this would be selecting all buttons and all inputs, please correct this to only set the person's name

Reply:
setTextToFieldNoEnter(NAME,name); allows the input of the string "john"